client/app/stories/Avatar.stories.jsx

```diff
@@ -1,21 +1,38 @@
+import { Col, Row } from 'antd';
 import React from 'react';
 import { withInfo } from '@storybook/addon-info';
 import { storiesOf } from '@storybook/react';
 
 import Avatar from 'bundles/shared/components/Avatar';
-import photo from 'app/assets/images/contributors/julia_nguyen.jpg';
+import photoJulia from 'app/assets/images/contributors/julia_nguyen.jpg';
+import photoTara from 'app/assets/images/contributors/tara_swenson.jpg';
+
+const withSource = el => withInfo()(() => el);
 
 storiesOf('Avatar', module)
-  .add('With name', withInfo({})(() =>
-    (<Avatar
-      src={photo}
-      alt="Julia Nguyen"
-      name="Julia Nguyen"
-    />),
+  .add('With name', withSource(
+    <Row style={{ backgroundColor: '#aaa' }}>
+      <Col span={8}>
+        <Avatar src={photoJulia} name="Julia &ldquo;Fleurchild&rdquo; Nguyen" displayname />
```

Comment:
LOL 🌷👧

Reply:
Haha, I'm just testing to see what the component looks like when the line length exceeds the width of the image. Honest!